adobe/react-spectrum
# fix(#2475): TextArea's input is smaller than the component's height property

Closes #2475

## ✅ Pull Request Checklist:

- [X] Included link to corresponding [React Spectrum GitHub Issue 2475](https://github.com/adobe/react-spectrum/issue/2475).

## 📝 Test Instructions:

1. open https://reactspectrum.blob.core.windows.net/reactspectrum/1f56f1e124efbd66e4bf6b7480b97bb2da2fae53/storybook/index.html?path=/story/textarea--resize-with-changeable-helptext&providerSwitcher-toastPosition=bottom
2. verify that textarea and helptext render to fill the `size-2000` height of `192px`
![Screen shot showing fixed height with for a TextArea with labelPosition: 'side' and a description that wraps](https://user-images.githubusercontent.com/154077/139910164-ec3939ff-d74f-4e8e-977b-330ebb85c4f8.png)
3. enter text in the field to trigger an error message. 
![Screen shot showing fixed height with for a TextArea with labelPosition: 'side' and an error message that doesn't wrap](https://user-images.githubusercontent.com/154077/139910620-18a90bc4-6b05-4a32-b0db-6ea026523e44.png)
4. notice the the textarea height adjust to fill the space when the helptext height changes, per flex box stying rules.
5. open https://reactspectrum.blob.core.windows.net/reactspectrum/1f56f1e124efbd66e4bf6b7480b97bb2da2fae53/storybook/index.html?path=/story/textarea--resize-with-changeable-helptext-custom-height&providerSwitcher-toastPosition=bottom
6. verify that the default textarea elements with size-2000 render to fill the space.
![Screen shot showing fix for TextArea components with custom heights](https://user-images.githubusercontent.com/154077/139910968-2837a5d9-7265-4614-85d6-054e303afa94.png)




## 🧢 Your Project:

Adobe/Accessibility
